ios/Podfile.lock

```diff
@@ -436,4 +436,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: fcf9d1ebed9e236314299522a2ebb30069ee9556
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.10.1
```

Comment:
Is this expected to have been downgraded?

Reply:
No, that's me running an older version. I will push a fix for this soon:tm: